### ISSUE TITLE
ApplicationConfiguration resource deleting, get applications api return nil

### ISSUE THREAD

**Issue by majian159** [oam-dev/kubevela#780]
**Describe the bug**

**To Reproduce**
up a app `vela.yaml`
```yaml
# vela.yaml
name: first-vela-app
services:
  testsvc:
    type: webservice
    image: crccheck/hello-world
    port: 8000
    route:
      domain: testsvc.example.com
```

```console
# up a app
vela up vela.yaml

# start dashboard api server
vela dashboard

# delete app and get apps
curl -w "\n" -X DELETE http://localhost:38081/api/envs/default/apps/first-vela-app ; curl -w "\n" http://localhost:38081/api/envs/default/apps/ ; sleep 3 && curl http://localhost:38081/api/envs/default/apps/

# output
{"code":200,"data":"delete apps succeed first-vela-app from default"}
{"code":200,"data":null}
{"code":200,"data":[{"name":"nginx-vela","status":"True","createdTime":"2020-12-11 16:47:07 +0800 CST"}]}
```

**Expected behavior**
not return nil

**Comment by majian159** [comment]
#779

**Comment by hongchaodeng** [comment]
fixed by #779
